support/agent/cases_test.go

```diff
@@ -240,6 +240,88 @@ func TestCreateDeals(t *testing.T) {
 	}
 }
 
+func TestCCUpgrades(t *testing.T) {
+	t.Skip("this is slow")
```

Comment:
Idea: not sure how to do it from go end but it would be awesome to create a make command that somehow runs these slow tests.  I think I remember we found a way to do this in gfc.